### DISCUSSION TITLE
Nginx Issue running from Docker

### DISCUSSION THREAD

**Discussion by cmjdev** [Budibase/budibase#11691] [Help]
I'm not able to run this as defined in the install documentation. My only change is not adding port information as I use an internal reverse proxy.

Here's the error. Any help is  much appreciated.

`ERROR: Budibase is not running 2023/09/06 22:32:00 [emerg] 441#441: socket() [::]:80 failed (97: Address family not supported by protocol) nginx: [emerg] socket() [::]:80 failed (97: Address family not supported by protocol) nginx: configuration file /etc/nginx/nginx.conf test failed ERROR: Nginx config problem`

**Comment by melohagan** [comment]
Hey @cmjdev 

It appears that it could be an issue with IPv6. Does this guide help at all? https://www.cyberciti.biz/faq/nginx-ipv6-configuration/

(scroll down to the section covering the `nginx.conf`)

If that isn't helpful, I can ask someone to reach out at some stage this week.
